app/views/home.html.php

```diff
@@ -64,46 +62,52 @@ function unicodeString($str, $encoding=null) {
             <?php } ?>
           </select>
         </div>
-      </div>
 
-      <div class="row">
         <div class="col-md-4">
-          <label for="filter-file-type">Input Type &nbsp;</label>
-          <select class="input-control" id="filter-file-type">
+          <label for="filter-input-type">Input Type &nbsp;</label>
+          <select class="input-control" id="filter-input-type">
             <option value="null"></option>
             <?php foreach ($fileTypes as $type) { ?>
               <option><?= $type ?></option>
             <?php } ?>
           </select>
         </div>
+      </div>
 
+      <div class="row">
         <div class="col-md-4">
-          <div class="input-group">
-            <label for="filter-name">Name &nbsp;</label>
-            <input type="text" class="input-control" id="filter-name" placeholder="Name..">
-          </div>
+          <label for="filter-name">Name &nbsp;</label>
+          <input type="text" class="input-control" id="filter-name" placeholder="Name..">
         </div>
 
         <div class="col-md-4">
-          <div class="input-group">
-            <label for="filter-last-state">Failure State &nbsp;</label>
-            <select class="input-control" id="filter-last-state">
-              <option value="null"></option>
-              <?php foreach (array_keys($ImportStates) as $state) { ?>
-                <option><?= $state ?></option>
-              <?php } ?>
-            </select>
+          <label for="filter-last-state">Failure State &nbsp;</label>
+          <select class="input-control" id="filter-last-state">
+            <option value="null"></option>
+            <?php foreach (array_keys($ImportStates) as $state) { ?>
```

Comment:
I prefer camel or snake to pascal case 
so $importStates or $import_states, not $ImportStates